### PR TITLE
Move Pagination to core

### PR DESCRIPTION
**Pull request by alycrys**

### CODE REVIEW

**Review by feliciackh — COMMENTED** [review]
site's docs will also need updating to core

File: .changeset/funny-windows-prove.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+"@salt-ds/core": patch
+"@salt-ds/data-grid": patch
+"@salt-ds/lab": patch
```
Comment (feliciackh) [inline .changeset/funny-windows-prove.md:4]: 🤔 is it `minor` instead?

Comment (joshwooding) [reply .changeset/funny-windows-prove.md:4]: Yep

Comment (alycrys) [reply .changeset/funny-windows-prove.md:4]: good question. @joshwooding ?

**Comment by joshwooding** [comment]
The Vercel build is failing with:

image

**Review by feliciackh — APPROVED** [review]

File: .yarn/patches/@jpmorganchase-mosaic-components-npm-0.1.0-beta.60-034892b5cb.patch [hunk 2]
```diff
@@ -1,7 +1,7 @@
-diff --git a/dist/chunk-ABJ2QGLD.js b/dist/chunk-ABJ2QGLD.js
-index 4154d3adbf29d771eeaf731a539bd1fdb2bcd4f3..d9a779da869932f91276bc5af09b812fb8e5ae80 100644
---- a/dist/chunk-ABJ2QGLD.js
-+++ b/dist/chunk-ABJ2QGLD.js
+diff --git a/dist/chunk-W3HASNBP.js b/dist/chunk-W3HASNBP.js
```
Comment (joshwooding) [inline .yarn/patches/@jpmorganchase-mosaic-components-npm-0.1.0-beta.60-034892b5cb.patch:1]: This patch has removed the Pill patch. Which is why the site build is failing

**Review by feliciackh — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**